src/stores/grid_renderer_store.ts

```diff
@@ -338,7 +345,15 @@ export class GridRenderer {
     const { ctx } = renderingContext;
     for (const box of boxes) {
       for (const icon of Object.values(box.icons)) {
-        if (!icon || !icon.svg) {
+        if (!icon) {
+          continue;
```

Comment:
the icon can be falsy ?

Reply:
![image](https://github.com/user-attachments/assets/385558c6-e428-4cc3-9599-a1927019bbea)

box.icons could be `{left: undefined, right: anIcon}` as per the typing. Not sure if it really happens in practice, but if I added the condition I guess it did at some point